Cypress/cypress/integration/Parallel_Tests/Create_Workflow/template.spec.js

```diff
@@ -87,7 +87,7 @@ describe("Testing the workflow creation wizard using Templates", () => {
 					.should("include.text", workflows.nonRecurringworkflowName); // Matching Workflow Name Regex
 				cy.wrap($div).find("td").eq(1).should("have.text", "Self-Agent"); // Matching Target Agent
 			});
-		cy.get("[data-cy=browseScheduleOptions]").eq(1).click();
+		cy.get("[data-cy=browseScheduleOptions]").click();
```

Comment:
We will need eq() here when we have multiple schedules. Although it should be eq(0) instead of eq(1) to get the dropdown of the latest schedule. @amityt

Reply:
Yes, it should be eq(0) if we have multiple schedules.